docs/Configuration.rst

```diff
@@ -6130,6 +6162,22 @@ Blockchain transaction settings
     .. note:: If the transaction is still not mined after ``exo.wallet.transaction.pending.maxDays`` days, it will be marked as a failed transaction in eXo's internal database.
               if/when it gets eventually mined (pending.maxDays), the Job ``ContractTransactionVerifierJob`` will detect it and will update the real transaction status coming from the blockchain and send notifications.
 
+* The parellel number of transactions to send:
+
+    .. code-block:: jproperties
+
+          exo.wallet.transaction.pending.maxToSend=3
+
+* The number of sending transaction attempts is configured by:
+
+    .. code-block:: jproperties
+
+          exo.wallet.transaction.pending.maxSendingAttempts=3
+
+
+    .. note:: If the transaction is still not mined after ``exo.wallet.transaction.pending.maxDays`` days, it will be marked as a failed transaction in eXo's internal database.
```

Comment:
```suggestion
    .. note:: If the transaction is still not mined after ``exo.wallet.transaction.pending.maxDays`` days, it will be marked as a failed in eXo's internal database.
```